wire-ios-request-strategy/Sources/Message Sending/SessionEstablisher.swift

```diff
@@ -18,28 +18,35 @@
 
 import Foundation
 
-public enum SessionEstablisherError: Error {
+public enum SessionEstablisherError: Error, Equatable {
     case missingSelfClient
     case networkError(NetworkError)
 }
 
-public class SessionEstablisher {
+// sourcery: AutoMockable
+public protocol SessionEstablisherInterface {
+    func establishSession(with clients: Set<QualifiedClientID>, apiVersion: APIVersion) async -> Swift.Result<Void, SessionEstablisherError>
```

Comment:
Since you are touching this now, couldn't you just remove `Result`?
```suggestion
    func establishSession(with clients: Set<QualifiedClientID>, apiVersion: APIVersion) async throws
```

Reply:
I did: https://github.com/wireapp/wire-ios/pull/648